CHANGELOG.md

```diff
@@ -3,6 +3,36 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Upcoming
+
+See also [2.0 roadmap](https://github.com/microsoft/MixedReality-WebRTC/issues/142).
+
+### Changed
+
+- (ba75e79c) Replace old track-based API with a transceiver-based API, allowing multiple audio and video tracks per peer connection. (#152)
+- (1a58b7be) Introduce proper standalone track source objects which can be shared among multiple tracks, including tracks from different peer connection. This allows sharing _e.g._ a single microphone or webcam track among multiple peer connections. (#412)
+- (46c058ae) Change the distribution model of the Unity library to use the Unity Package Manager (UPM). This allows better encapsulation and easier installation for users, as well as enabling distributing prebuilt packages including all Windows and Android binaries.
+
+### Fixed
+
+- (25058231) Fix compatiblity with other WebRTC implementation using non-numeric MID numbers in SDP messages. (#295)
+- (b7caf9c9) Recycle ARGB buffer when frame rotates (_i.e._ landscape vs. portrait). (#435)

```

Comment:
What is the criteria for this list? I was under the impression that we fixed more stuff 😄

Reply:
We definitely did. Lack of motivation for listing all small stuffs, and realizing that a lot of things we fixed are from bugs we introduced in the same iteration, so are not relevant for an incremental changelog v1.0.3 -> v2.0.0.

Reply:
If you have some more interesting ones in mind we can add them. That's the ones I found scanning through the 400+ commits since the `v1.0.3` tag.